cypress/e2e/cloud/home.test.ts

```diff
@@ -39,25 +39,28 @@ describe('Home Page Tests', () => {
     createChecks()
     exportMockAlertsActivity()
 
+    // Random amount of wait for the export data to be available on home page
+    cy.wait(1000)
+
     cy.visit('/')
     cy.getByTestID('alerts-activity')
       .scrollIntoView()
       .should('be.visible')
     cy.getByTestID('alerts-activity-table-container')
       .find('.event-row')
-      .should('have.length', 2)
+      .should('have.length.gte', 2)
```

Comment:
what's with the `gte`?

Reply:
Greater Than Equal to. This is so that if the checks have been run and the data is populated, then there would be 3-4 rows including the 2 of mock ones.

Reply:
TIL